contracts/honeylemon/MarketContractProxy.sol

```diff
@@ -545,13 +546,13 @@ contract MarketContractProxy is Ownable {
         shortToken.transfer(getUserAddressOrDSProxy(shortTokenRecipient), qtyToMint);
 
         emit PositionTokensMinted(
+            qtyToMint,
             addressToMarketId[address(latestMarketContract)], // MarketID
```

Comment:
Modified the order here in the event to fix the `CompilerError: Stack too deep, try removing local variables.` error.

Reply:
this change will break the graph implementation @andrei-anisimov. We will need to update that accordingly when this is merged in.